src/workspacer/workspacer.csproj

```diff
@@ -43,11 +49,16 @@
   <Target Name="CopyPluginsAfterPublish" AfterTargets="Publish">
     <Copy SourceFiles="$(SolutionDir)src\workspacer.Bar\$(OutDir)workspacer.Bar.dll" DestinationFolder="$(PublishDir)plugins\workspacer.Bar\" />
     <Copy SourceFiles="$(SolutionDir)src\workspacer.Gap\$(OutDir)workspacer.Gap.dll" DestinationFolder="$(PublishDir)plugins\workspacer.Gap\" />
+	<Copy SourceFiles="$(SolutionDir)src\workspacer.FocusBorder\$(OutDir)workspacer.FocusBorder.dll" DestinationFolder="$(PublishDir)plugins\workspacer.FocusBorder\" />
```

Comment:
Weird indentation here with a mixture of tabs and spaces. Keep to spaces only please :)